src/server/editorServices.ts

```diff
@@ -46,6 +46,8 @@ namespace ts.server {
 
 /* __GDPR__
    "projectInfo" : {
+        "${include}": ["${TypeScriptCommonProperties}"],

```

Comment:
Did we figure out how this interacts with `version` below?

Reply:
The included one overrides the local one. I think I'll leave it in since TypeScriptCommonProperties could (in theory) also *remove* properties over time as well as add more.

Reply:
Weird that the local one wouldn't win, but I agree that keeping both makes sense.